### DISCUSSION TITLE
After Wikilinks a String appears in curly brackets

### DISCUSSION THREAD

**Discussion by vamonk** [mkdocs-publisher/mkdocs-publisher#102] [Q&A] [answered]
How can I prevent this display? 

![image](https://github.com/user-attachments/assets/0381b19c-0c53-4da0-999b-b15d8d1a0721)

**Comment by vamonk** [comment, marked as answer]
#101 solved it.

**Reply by mkusz** [reply]
I will add information about it to the documentation and then close this discussion.
